2.1.0/docker-entrypoint.sh

```diff
@@ -36,6 +36,9 @@ if [ "$1" = '/opt/couchdb/bin/couchdb' ]; then
 
 	if [ ! -z "$NODENAME" ] && ! grep "couchdb@" /opt/couchdb/etc/vm.args; then
 		echo "-name couchdb@$NODENAME" >> /opt/couchdb/etc/vm.args
+    else
```

Comment:
Indentation of this section looks wrong.

Reply:
![image](https://user-images.githubusercontent.com/158641/31349228-aae9cc04-ad22-11e7-8587-af55f962f2d6.png)
Damn tabs vs spaces....